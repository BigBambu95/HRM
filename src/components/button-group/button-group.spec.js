import React from 'react'
import ButtonGroup from './button-group'
import Button from '../button/button'
import { render } from 'enzyme'

describe('Группа кнопок', () => {
	const setUp = (props) =>
		render(
			<ButtonGroup {...props}>
				<Button>Click 1</Button>
				<Button>Click 2</Button>
			</ButtonGroup>
		)

	it('Рендер группы кнопок', () => {
		const buttonGroup = setUp()
		expect(buttonGroup).toMatchSnapshot()
	})

	it('Вертикальная группа кнопок', () => {
		const buttonGroup = setUp({ vertical: true })
		expect(buttonGroup).toMatchSnapshot()
	})
})
