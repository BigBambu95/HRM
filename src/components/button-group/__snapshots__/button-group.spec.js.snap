// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Группа кнопок Вертикальная группа кнопок 1`] = `
<div
  class="button-group vertical"
>
  <button
    class="btn medium outlined purple"
    type="button"
  >
    Click 1
  </button>
  <button
    class="btn medium outlined purple"
    type="button"
  >
    Click 2
  </button>
</div>
`;

exports[`Группа кнопок Рендер группы кнопок 1`] = `
<div
  class="button-group"
>
  <button
    class="btn medium outlined purple"
    type="button"
  >
    Click 1
  </button>
  <button
    class="btn medium outlined purple"
    type="button"
  >
    Click 2
  </button>
</div>
`;
